### PR TITLE
Bluespace turfs no longer emit light.

### PR DESCRIPTION
**Pull request by Hockaa**
Removes the light_color, light_power, light_range and dynamic_lighting vars from bluespace turfs.
Removes initialize()

Requested by Geeves.

### CODE REVIEW

**Review by Geevies — APPROVED** [review]

File: code/game/turfs/space/transit.dm [hunk 1]
```diff
@@ -6,14 +6,9 @@
 	plane = 0
 	use_space_appearance = FALSE
 	use_starlight = FALSE
-	light_color = COLOR_CYAN_BLUE
-	light_power = 6
-	light_range = 8
-	dynamic_lighting = TRUE
 
 /turf/space/transit/Initialize()
```
Comment (Geevies) [inline code/game/turfs/space/transit.dm:10]: you can remove this part as well

**Comment by Hockaa** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by fernerr — CHANGES_REQUESTED** [review]
I need a proper explanation of why this is being removed when I spent a lot of time in the new shuttle update getting this to work properly with all the shuttle edges. Lighting updates take a little bit on transporting to new areas, but I don't feel that is reason enough.

**Review by fernerr — APPROVED** [review]
Reason is startup lag, particularly for the lighting system. Might look into adding a less demanding version at some point.

**Comment by NonQueueingMatt** [comment]
We can probably do something similiar by just finding every mob in the shuttle and applying a blue filter to their view @fernerr @Geevies

**Pull request merged**